.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,24 @@
+# Release Description
+<!--- Summarize what is being released.  -->
+
+# Release Checklist
+<!--- Fill out the following Release checklist -->
+- [ ] Version number has been updated
+- [ ] Description file updated with New Developers (if applicable)
+- [ ] NEWS.md has been updated and issues linked where appropriate
+- [ ] Ensure all unit tests are passing and Test Coverage is Green on Readme
```

Comment:
It shouldn't be possible to merge into main if any unit test is failing. So should we remove "all unit tests are passing and"?

Reply:
I guess I just want us to have that Test Coverage above 90% and a green badge on the Readme.  

I'll change it to just the Test Coverage Badge is green on the Readme